webui/react/src/e2e/fixtures/api.workspace.fixture.ts

```diff
@@ -52,25 +32,21 @@ export class ApiWorkspaceFixture {
    * strict superset of the Response, so no info is lost.
    */
   async createWorkspace(req: V1PostWorkspaceRequest): Promise<V1PostWorkspaceResponse> {
-    const apiAuth = this.apiAuth;
-    const workspaceResp = await (await this.startWorkspaceRequest())
-      .postWorkspace(req, {})
-      .catch(async function (error) {
-        const respBody = await streamConsumers.text(error.body);
-        if (error.status === 401) {
-          const token = apiAuth.getBearerToken();
-          throw new Error(
-            `Create Workspace Request failed. Status: ${error.status} Request: ${JSON.stringify(
-              req,
-            )} Token: ${token} Response: ${respBody}`,
-          );
-        }
+    const workspaceResp = await this.api.postWorkspace(req, {}).catch(async (error) => {
+      const respBody = await streamConsumers.text(error.body);
+      if (error.status === 401) {
```

Comment:
q: why does a 401 Unauthorized error get a special error message?

Reply:
I assume that it's to allow for checking the user token after an error.